api/src/test/java/com/epam/pipeline/manager/pipeline/PipelineConfigurationManagerTest.java

```diff
@@ -16,182 +16,193 @@
+                        return nfsDs.getMountOptions().equals(option) ||
+                                option.equals(nfsDs.getMountOptions() + ",ro");
+                    }));
         }
 
-        String[] mountPoints = config.getMountPoints().split(";");
+        String[] mountPoints = deleteEmptyElements(config.getMountPoints().split(";"));
         for (String mountPoint : mountPoints) {
-            if (StringUtils.isNotBlank(mountPoint)) {
-                Assert.assertTrue(dataStorages.stream().anyMatch(ds -> mountPoint.equals(ds.getMountPoint())));
-            }
+            assertTrue(dataStorages.stream().anyMatch(ds -> mountPoint.equals(ds.getMountPoint())));
         }
         //Assert.assertTrue(Arrays.stream(nfsOptions).anyMatch(o -> o.endsWith(",ro")));
+
+        assertFalse(config.isNonPause());
+        assertEquals(TEST_INSTANCE_IMAGE, config.getInstanceImage());
+        assertEquals(TEST_INSTANCE_TYPE, config.getInstanceType());
+        assertEquals(TEST_PRETTY_URL, config.getPrettyUrl());
+        assertEquals(TEST_WORKER_CMD, config.getWorkerCmd());
+        assertEquals(TEST_CMD_TEMPLATE, config.getCmdTemplate());
+        assertEquals(TEST_NODE_COUNT, config.getNodeCount());
+        assertEquals(TEST_HDD_SIZE, Integer.valueOf(config.getInstanceDisk()));
+        assertTrue(config.getIsSpot());
+        assertEquals(TEST_PARAMS, config.getParameters());
+        assertEquals(TEST_REPO + "/" + TEST_IMAGE, config.getDockerImage());
+        assertEquals(TEST_TIMEOUT, config.getTimeout());
+
+        verify(pipelineVersionManager).getValidDockerImage(eq(TEST_IMAGE));
+        verify(dataStorageApiService).getWritableStorages();
+        verify(permissionsService, times(2)).isMaskBitSet(
+                eq(AbstractDataStorage.ALL_PERMISSIONS_MASK),
+                eq(((AclPermission) AclPermission.WRITE).getSimpleMask()));
     }
 
     private PipelineStart getPipelineStartVO() {
         PipelineStart vo = new PipelineStart();
+        vo.setNonPause(TEST_NON_PAUSE);
+        vo.setInstanceImage(TEST_INSTANCE_IMAGE);
+        vo.setPrettyUrl(TEST_PRETTY_URL);
+        vo.setWorkerCmd(TEST_WORKER_CMD);
         vo.setInstanceType(TEST_INSTANCE_TYPE);
         vo.setDockerImage(tool.getImage());
-        vo.setHddSize(1);
-        vo.setCmdTemplate("template");
+        vo.setHddSize(TEST_HDD_SIZE);
+        vo.setCmdTemplate(TEST_CMD_TEMPLATE);
         vo.setTimeout(TEST_TIMEOUT);
-        vo.setNodeCount(1);
-        vo.setIsSpot(true);
-        vo.setParams(Collections.singletonMap("testParam", new PipeConfValueVO("testParamValue", "int", true)));
+        vo.setNodeCount(TEST_NODE_COUNT);
+        vo.setIsSpot(TEST_IS_SPOT);
+        vo.setParams(TEST_PARAMS);
         return vo;
     }
+
+    private String[] deleteEmptyElements(String[] array) {
+        return Arrays.stream(array)
+                .filter(StringUtils::isNotBlank)
+                .toArray(String[]::new);
+    }
 }
```

Comment:
Could you please add an empty line at the end of the file?

Reply:
Fixed